options/locale/locale_en-US.ini

```diff
@@ -913,7 +913,7 @@ settings.wiki_deletion_success = Repository wiki data have been erased.
 settings.delete = Delete This Repository
 settings.delete_desc = Once you delete a repository, there is no going back. Please be certain.
 settings.delete_notices_1 = - This operation <strong>CANNOT</strong> be undone.
-settings.delete_notices_2 = - This operation will permanently delete everything in the <strong>%s</strong> repository, including code, issues, comments, the wiki, and collaborators associations.
+settings.delete_notices_2 = - This operation will permanently delete everything in the %s repository, including code, issues, comments, the wiki, and collaborators associations.
```

Comment:
This locale is used in multiple places, so I don't think removing the `<strong>` tags is the right thing to do.

Apologies if my previous comment was not clear, but if we now plan to continue rendering with `<strong>` tags, changing the locale is not necessary.

Reply:
Also, if we add the `<strong>` tags back, maybe it'd be best to add a class (e.g. `<strong class="repo-name">%s</string>`) so we can just select/modify/remove the repo name using JQuery (which would make the Javascript a little cleaner).

Reply:
agreed, keep the `<strong>` in locale, and only expect to find `%s` when you replace later

Reply:
okay, i reverted the strong tag